index.bs

```diff
@@ -1653,6 +1653,10 @@ To <dfn export>process an attribution eligible response</dfn> given a [=suitable
     state is not immediately detectable. Attribution may also be blocked if the reporting origin is not
     <a href="https://github.com/privacysandbox/attestation">enrolled</a>.
 
+1. [=Queue a task=] on the [=networking task source=] to proceed with the following steps.
```

Comment:
It seems that these tasks are more related to dealing with the event queue which I'm not sure is exactly what we need here. I had in mind to look at these steps: https://html.spec.whatwg.org/multipage/infrastructure.html#parallelism i.e., creating a new parallel queue and en queuing on it.

Did you look at both, could you detail the idea ? thanks

Reply:
I think these tasks should be run in the same queue to ensure the sequence, so using the event queue seems to be better?

Reply:
ack. I read further on the event queue, that makes sense, thanks.